packages/instantsearch.js/src/widgets/hits/__tests__/hits-integration-test.ts

```diff
@@ -122,7 +122,101 @@ describe('hits', () => {
       );
     });
 
-    it('sends click event', async () => {
+    it('sendEvent sends click event', async () => {
+      const { search } = createInstantSearch();
+      const { insights, onEvent } = createInsightsMiddlewareWithOnEvent();
+      search.use(insights);
+
+      search.addWidgets([
+        hits({
+          container,
+          templates: {
+            item: (item, { html, sendEvent }) => html`
+              <button
+                type="button"
+                onClick=${() => sendEvent('click', item, 'Item Clicked')}
+              >
+                ${item.title}
+              </button>
+            `,
+          },
+        }),
+      ]);
+      search.start();
+      await wait(0);
+
+      expect(onEvent).toHaveBeenCalledTimes(1); // view event by render
+      fireEvent.click(getByText(container, 'title 1'));
+      expect(onEvent).toHaveBeenCalledTimes(2);
+      expect(onEvent.mock.calls[onEvent.mock.calls.length - 1][0]).toEqual({
```

Comment:
Can we clear the mock before so we remove the view event and we can simplify this assertion with `toHaveBeenCalledWith`?